server/db/index.js

```diff
@@ -0,0 +1,19 @@
+const mysql = require('mysql');
+const createTables = require('./config');
+const Promise = require('bluebird');
+const database = 'TREK';
+
```

Comment:
You may want to drop the database before creating it, so it resets between server resets. I had an issue where it wouldn't add reviews until the database was dropped manually, then recreated by restarting the server.